### DISCUSSION TITLE
Wiz Recessed Downlight

### DISCUSSION THREAD

**Discussion by BenQuinlan** [sbidy/wiz_light#190] [Q&A] [answered]
I've been researching color LED recessed lighting to install and found this (amazing) HA integration. I wanted to ask if anyone has any experience using this with the Wiz Recessed Downlights, and if there have been any issues? My use case would be to change the color of all the recessed lighting via home assistant.

**Comment by martroy** [comment, marked as answer]
I have ten of the recessed downlights and I can confirm that everything works great controling them through HA and this integration. I highly recommend them.

**Reply by BenQuinlan** [reply]
Thanks for the clarification, just wanted to double check before I purchased a bunch! Did you ever consider using the recessed Hue lights instead of Wiz? If so, what made you decide on Wiz?

**Reply by martroy** [reply]
I chose the Wiz due the fact that they don't need a gateway and the price compared to the Hue ones (+ I got them at 60% off last year).
